### PR TITLE
Prettier player

### PR DESCRIPTION
**Pull request by jjenzz**
- Fixes the "In the media" links on the homepage
- Fixes code warnings
- Updates player with better designed one

**NOTE**: Player isn't working at the moment because the `audioUrl` returned from [API here](https://musicoin.org/json-api/track/0x859dc166c7c2a87bae4ebca55daae166cde3d59d) does not work. @Varunram Can you help with this tomorrow please?

**BEFORE**

image

**AFTER**

image

### CODE REVIEW

File: src/Home/index.js [hunk 1]
```diff
@@ -35,23 +29,27 @@ export const Home = () => (
...
 			</Content>
 			<section>
 				<Content>
+					<SecondaryHeading level={2}>Artist of the week</SecondaryHeading>
+					<Player
+						artist="IXIISIS"
+						title="Vertigo"
+						track="/ppp/38fb0550a162cd13e5d749fd9977f7e6f19532fbbfdeec469b15dcd81392eeab8eced6b6fcbac9ca89268fb5f66894c9b5bcff5fc326e032"
+						coverSrc="/media/cf392d93aff3f8147aa516d4da0e12e1ff5bd52317037d982034a18aa1ad1f80574c506190aa012e3d4f7c2e895b"
+						tipCount={1092}
```
Comment (jjenzz) [inline src/Home/index.js]: Hardcoded for now for release

File: src/shared/Player/styles.js [hunk 2]
```diff
@@ -1,50 +1,147 @@
 import React from 'react';
 import styled from 'styled-components';
```
Comment (jjenzz) [inline src/shared/Player/styles.js]: This file was copied from wannameows changes. I haven't had a chance to review it properly but we can do that later.

**Comment by Varunram** [comment]
I tried streaming directly from the player URL, seems like it doesn't work without the backend routes. Maybe we can look at this in MVP2, I styled the current player a bit and it looks half decent now.

**Comment by jjenzz** [comment]
@Varunram why was this merged if it doesn't work? 😛

**Comment by Varunram** [comment]
I built upon the commits of this PR and GitHub automatically closes PRs with past commit history :)

**Comment by cryptofuture** [comment]
@jjenzz New player exactly didn't worked and blocked feed rendering, so I rolled back to old site version. Gladly all this took around 10 seconds.

**Comment by jjenzz** [comment]
@cryptofuture Yes, I know the new player didn't work. It says that in my PR description which is why it should not have been merged 🙂

**Pull request merged**